src/transformers/models/m2m_100/modeling_m2m_100.py

```diff
@@ -1244,12 +1246,8 @@ class M2M100ForConditionalGeneration(M2M100PreTrainedModel):
         r"encoder.version",
         r"decoder.version",
         r"lm_head.weight",
-        r"model.encoder.embed_positions.weights",
```

Comment:
I realized these are removed, is this intended (I see that it's one of the only model where some keys are removed)? Maybe it has been done by mistake but I am not sure

Reply:
Seems I forgot to re-add this. 

This was removed because of the test which is now commented which tries to remove `_key_to_ignore` which are masking real weights.

This SinusoidalPositionEmbeddings, seems to be a buffer and not a parameter, but something is wrong somewhere.
Since it's a nn.Module, maybe it still sees  `weights` as being a Parameter somehow, idk.

We can safely revert this part anyway.

Reply:
Thanks a lot for the explanation ! 🚀